tests/test_config_parser.py

```diff
@@ -150,7 +155,7 @@ def add_reference_config(self, job_name=DEFAULT_JOB_NAME):
             "workspace_dir": None,
         }
 
-    def __eq__(self, other: Mapping[str, Any]) -> bool:
+    def __eq__(self, other: Mapping[str, Any]) -> bool: # type: ignore
```

Comment:
This one was tricky. I tried doing it like [here](https://stackoverflow.com/questions/54801832/mypy-eq-incompatible-with-supertype-object):

```
def __eq__(self, other: object) -> bool:
    if not isinstance(other, Mapping[str, Any]):
        return NotImplemented
        [...]
```

But then Mypy says:
`error: Parameterized generics cannot be used with class or instance checks  [misc]`